recipes/libsequence/1.8.4/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version="1.8.4" %}
+{% set hash="24ac822e202549175aafbeb8db8dc5daa11e3874334fc9e55d0bb4b9bc81b221" %}
+
+package:
+  name: libsequence
+  version: {{ version }}
+
+source:
+  fn:  {{ version }}-1.tar.gz 
+  url: http://github.com/molpopgen/libsequence/archive/{{ version }}-1.tar.gz
+  sha256: {{ hash }}
+
+build:
+  number: 0
+  string: "htslib{{CONDA_HTSLIB}}_{{PKG_BUILDNUM}}"
```

Comment:
boost should be also added here

Reply:
Boost was a dependency of the `make check` step that keeps falling on macos x.